CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ This release modifies the behavior of setup.py with respect to dependencies.
 Previously, `boto3` and other AWS-related packages were installed by default.
 Now, in order to install them, you need to run either:
 
-    pip install smart_open[s3]
+    pip install smart_open[aws]
```

Comment:
Is this right?

Reply:
Ah, yes, this particular part we should _not_ revert. Good catch.